docs/rules/new-cap.md

```diff
@@ -135,14 +135,21 @@ function foo(arg) {
 
 ### capIsNewExceptionPattern
 
-Examples of additional **correct** code for this rule with the `{ "capIsNewExceptionPattern": "^Person\.." }` option:
+Examples of additional **correct** code for this rule with the `{ "capIsNewExceptionPattern": "^person\.." }` option:
 
 ```js
-/*eslint new-cap: ["error", { "capIsNewExceptionPattern": "^Person\.." }]*/
+/*eslint new-cap: ["error", { "capIsNewExceptionPattern": "^person\.." }]*/
 
 var friend = person.Acquaintance();
 var bestFriend = person.Friend();
 ```
+Examples of additional **correct** code for this rule with the `{ "capIsNewExceptionPattern": "\.Bar$" }` option:
+
+```js
+/*eslint new-cap: ["error", { "capIsNewExceptionPattern": "\.Bar$" }]*/
+
+foo.Bar(); // Lint-Free!
```

Comment:
There is no need for `// Lint-Free!`, this is already an example of correct code.

Also, can we please add a similar example for `newIsCapExceptionPattern`?